hibike/hibike_message_old.py

```diff
@@ -0,0 +1,234 @@
+from __future__ import print_function
```

Comment:
Why is this file part of this pull request? The name suggests that it is obsolete and can be removed.

(I'm not further code reviewing the contents of this file)